core/src/main/java/org/ldaptive/transport/netty/NettyConnection.java

```diff
@@ -818,17 +821,21 @@ protected void write(final DefaultOperationHandle handle)
                 bindLock.readLock().unlock();
               }
             } else {
-              handle.exception(new LdapException("Bind in progress"));
+              handle.exception(new LdapException(ResultCode.LOCAL_ERROR, "Bind in progress"));
```

Comment:
This one caught my eye because of err=14 (SASL bind in progress), but I think it makes sense in this case after looking at it closer.